### PR TITLE
Update rolling_update.md

### PR DESCRIPTION
**Pull request by rj03hou**
modify the document, add the checks before rolling.

### CODE REVIEW

**Comment by chrisz100** [comment]
@rj03hou its not your fault but can you please run make apimachinery as another commit? Your upstream seems to have outdated apimachinery information.

**Comment by rj03hou** [comment]
@chrisz100 how to do "run make apimachinery", I am a beginner to this.

File: docs/development/rolling_update.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Upgrading and modifying a k8s cluster often requires the replacement of nodes.  To not cause loss of service and other disruptions a kops uses a functionality call rolling updates.  Rolling a cluster is the replacement of the masters and nodes with new cloud instances.
 
+Before rolling update, kops will check whether the instance in the ig need to be update, so when you just update nodes, the master will not roll, when you rolling be interrupted, you run again, will not roll the new instance.
```
Comment (Pharb) [inline docs/development/rolling_update.md]: Thanks for improving the documentation, maybe this addition can be phrased a bit differently:

```suggestion
When starting the rolling update, kops will check each instance in the instance group if it needs to be updated, so when you just update nodes, the master will not be updated. When your rolling update is interrupted and you run another rolling update, instances that have been updated before will not be updated again.
```

**Comment by Pharb** [comment]
> @chrisz100 how to do "run make apimachinery", I am a beginner to this.

@rj03hou I think @chrisz100 literally means running `make apimachinery` as [documented here](https://github.com/kubernetes/kops/blob/master/docs/development/api_updates.md#updating-the-generated-api-code).

Could you instead please first try to rebase your branch to the lastet upstream master, push that to your branch and see if the `continuous-integration/travis-ci/pr` check in this pull request becomes green?

**Comment by justinsb** [comment]
Thanks @rj03hou  - nice clarification of how this (should!) work

/approve
/lgtm

**Pull request merged**